src/Extensions/SwiftypeFileCrawlerExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Ichaber\SSSwiftype\Extensions;
+
+use SilverStripe\Assets\File;
+
+/**
+ * @property File|$this $owner
+ */
+class SwiftypeFileCrawlerExtension extends SwiftTypeCrawlerExtension
+{
+    /**
+     * Method to check our file types whitelist since we don't want to index files that aren't required in the index
+     * e.g. image files.
+     */
+    protected function recordCanBeIndexed(): bool
+    {
+        // only reindex file types we need.
+        $fileType = File::get_file_extension($this->getOwner()->Filename);
+
+        return in_array($fileType, $this->getOwner()->config()->get('reindex_files_whitelist'), true);
```

Comment:
Not to be overly woke, but I'd prefer the use of "allow" over "white". It's kind of an easy thing to change to be [doing our part]. This comment would go for the `README.md`, tests, etc.

This doesn't have to be added now (it can be an improvement later), but it would also be good to have support for file categories ("images", "documents", etc).

Reply:
I definitely prefer allowlist as well, will change